tox.ini

```diff
@@ -33,19 +32,22 @@ commands =
 # we separate coverage into its own testenv because a) "last run wins" wrt
 # cobertura jenkins reporting and b) pypy and jython can't handle any
 # combination of versions of coverage and nosexcover that i can find.
-[testenv:py2-cover]
+
+[py-cover]
```

Comment:
This is fine to factor out... however this tox.ini is basically carbon copied across several projects right now so forking it for a minor improvement feels a little... meh.